docs/index.md

```diff
@@ -4,11 +4,33 @@ hide:
 - toc
 ---
 
-![logo](imgs/homelogo.png){align=left}
+![logo](imgs/homelogo.png)
 
 <br/>
 <br/>
 
+???+ warning
+     📣 All of our scanning-related efforts have moved to one place, and that is Trivy. Please look at the [migration guide](https://aquasecurity.github.io/trivy/latest/tutorials/tfsec/migration/) and the Trivy [docs](https://aquasecurity.github.io/trivy/latest/) for further information.
+
+## 📣 tfsec to Trivy Migration
+
+As part of our goal to provide a comprehensive open source security solution for all, we have been consolidating all of our scanning-related efforts in one place, and that is [Trivy](https://github.com/aquasecurity/trivy). 
```

Comment:
sounds too similar to previous para. I would keep this one and remove prev

Reply:
the idea was that the previous paragraph serves as a summary & is highlighted so people actually look at it when opening the docs but will remove it